app/src/main/java/nyc/c4q/jonathancolon/inContaq/utlities/graphs/WeeklyGraph.java

```diff
@@ -0,0 +1,240 @@
+        sentValues = setValues(getMonthlySent(lstSms));
+    }
+
+    private float[] setValues(TreeMap<Integer, Integer> numberOfTexts) {
+
+        ArrayList<Float> list = new ArrayList<Float>();
+        for (Map.Entry<Integer, Integer> entry : numberOfTexts.entrySet()) {
+
+            Float value = entry.getValue().floatValue();
+            list.add(value);
+        }
+        return convertFloats(list);
+    }
+
+    private TreeMap<Integer, Integer> getDailyReceived(ArrayList<Sms> texts) {
+
+        TreeMap<Integer, Integer> monthlyReceived = setUpDailyTextMap();
+        MonthlyTaskParams receivedParams = new MonthlyTaskParams(texts, monthlyReceived);
+        MonthlyReceivedWorkerTask monthlyReceivedTask = new MonthlyReceivedWorkerTask();
+
+        try {
+
+            monthlyReceived = monthlyReceivedTask.execute(receivedParams).get();
+            Log.e(TAG, MONTHLY_RECEIVED + monthlyReceived);
+
+        } catch (InterruptedException | ExecutionException e) {
+
+            e.printStackTrace();
+        }
+        return monthlyReceived;
+    }
+
+    private TreeMap<Integer, Integer> getMonthlySent(ArrayList<Sms> texts) {
+
+        TreeMap<Integer, Integer> monthlySent = setUpDailyTextMap();
+        MonthlyTaskParams sentParams = new MonthlyTaskParams(texts, monthlySent);
+        MonthlySentWorkerTask monthlySentWorkerTask = new MonthlySentWorkerTask();
+
+        try {
+
+            monthlySent = monthlySentWorkerTask.execute(sentParams).get();
+            Log.e(TAG, MONTHLY_SENT + monthlySent);
+
+        } catch (InterruptedException | ExecutionException e) {
+            e.printStackTrace();
+        }
+        return monthlySent;
+    }
+
+    // This converts the generic "Float" to primitive float because the graphs only uses primitives
+    private float[] convertFloats(List<Float> floats) {
+
+        float[] ret = new float[floats.size()];
+        Iterator<Float> iterator = floats.iterator();
+        for (int i = 0; i < ret.length; i++) {
+            ret[i] = iterator.next().intValue();
+        }
+        return ret;
+    }
+
+    private TreeMap <Integer, Integer> setUpDailyTextMap() {
+
+        TreeMap<Integer, Integer> weeklyMap = new TreeMap<>();
+
+        weeklyMap.put(SUN, DEFAULT_VALUE);
+        weeklyMap.put(MON, DEFAULT_VALUE);
+        weeklyMap.put(TUE, DEFAULT_VALUE);
+        weeklyMap.put(WED, DEFAULT_VALUE);
+        weeklyMap.put(THUR, DEFAULT_VALUE);
+        weeklyMap.put(FRI, DEFAULT_VALUE);
+        weeklyMap.put(SAT, DEFAULT_VALUE);
+
+        return weeklyMap;
+    }
+
+    private void getHigehstValueForYaxis() {
+
+        highestValue = getYValue(sentValues, receivedValues);
+    }
+
+    synchronized private int getYValue(float[] sentValues, float[] receivedValues) {
+        int maxSent = findMaximumValue(sentValues);
+        int maxReceived = findMaximumValue(receivedValues);
+        if (maxSent > maxReceived) {
+            return highestValue = (int) getRound(maxSent);
```

Comment:
You should either return the number or set it to highestValue. You don't need to do both.